docs/introduction/getting-started.md

```diff
@@ -6,74 +6,133 @@ order: 400
 Getting started with Convox is easy. The instructions below guide you through:
```

Comment:
This file is the big set of changes. Everything else is just broken links etc...